data/net/minecraft/world/phys/shapes/BooleanOp.mapping

```diff
@@ -0,0 +1,52 @@
+CLASS net/minecraft/world/phys/shapes/BooleanOp
```

Comment:
This class is already mapped by https://github.com/ParchmentMC/Parchment/pull/48 and I also left a comment https://github.com/ParchmentMC/Parchment/pull/48/files#r676217878 (I think left + right are better than first + second given the connotations of left hand side <op> right hand side

To avoid unnecessary conflicts I think the easiest would be to drop this from this PR.

Reply:
I didn't intend to commit these here, I'll just drop all `net.minecraft.world.phys` mappings from this PR.